mass-deployer/pkg/mass-deployer/helpers.go

```diff
@@ -0,0 +1,17 @@
+package deployer
```

Comment:
Don't name it with helpers, Thabet doesn't like it :D you can add it with deployer file or have a file for each o_o

Reply:
😆 😆